Code/Framework/AzFramework/AzFramework/Viewport/CameraInput.cpp

```diff
@@ -23,6 +25,18 @@ namespace AzFramework
         AZ::ConsoleFunctorFlags::Null,
         "Should the camera use cursor absolute positions or motion deltas");
 
+#if AZ_TRAIT_AZFRAMEWORK_ENABLE_DISABLE_MOUSE_CAPTURE_FOR_CAMERA_CVAR_OPTION
+    AZ_CVAR(
+        bool,
+        ed_disable_capture_mouse_for_camera_rotation,
```

Comment:
```suggestion
ed_mouse_capture_for_camera_rotation
```
CVAR is a bool so doesn't need "disable" in the name. Also suggest starting with "ed_mouse" since mouse/cursor is probably what people would search.

Reply:
1. It can go either way, I prefer disable in this case because forces an active action to correct an uncommon situation that this works around. This isn't the only disable cvar, BTW
2. People will more likely search 'mouse' than 'ed_mouse', so it doesn't matter
3. The spacing is to be consistent with the cvar that was previously declared here